natives/lwjgl.js

```diff
@@ -1009,13 +1015,19 @@ function Java_org_lwjgl_opengl_GL11_nglNormalPointer(lib, type, stride, memPtr,
 	normalData.pointer = Number(memPtr);
 }
 
-function Java_org_lwjgl_opengl_GL13_nglMultiTexCoord2f()
+function Java_org_lwjgl_opengl_GL13_nglMultiTexCoord2f(lib, target, s, t)
 {
 	checkNoList(curList);
 	if(verboseLog)
+	{
 		console.log("glMultiTexCoord2f");
+		console.log("glMultiTexCoord2f(", target, ",", s, ",", t,")");
+			//i dont know what to do with these values}
+			//i keep getting these 2 values ( 33985 , 61680 , 0 ) and ( 33985 , 240 , 0 )
```

Comment:
These are texture coordinates. What they actually are depends on what the texture is at the time, i.e. the model that's being rendered.